toolkit/themes/shared/design-system/config.js

```diff
@@ -5,7 +5,25 @@
 /* eslint-env node */
 
 const StyleDictionary = require("style-dictionary");
-const { formattedVariables } = StyleDictionary.formatHelpers;
+const { createPropertyFormatter } = StyleDictionary.formatHelpers;
+
+const TOKEN_SECTIONS = {
+  "Base tokens/Color": [/^color-[^-]*$/, /^color-.*-a?\d+$/],
+  "Application tokens/Border": "border",
+  "Application tokens/Box": "box",
+  "Application tokens/Color": "color",
+  "Application tokens/Font weight": "font-weight",
+  "Application tokens/Focus outline": "focus-outline",
+  "Application tokens/Icon": "icon",
+  "Application tokens/Input/Button": "button",
+  "Application tokens/Input/Checkbox": "checkbox",
+  "Application tokens/Input/Text": "input-text",
+  "Application tokens/Link": "link",
+  "Application tokens/Text": "text",
+  "Application tokens/Size": "size",
+  "Application tokens/Spacing": ["space", "spacing"],
```

Comment:
```suggestion
  "Application tokens/Space": "space"
```

Should we just commit to using `space`? I don't see any `spacing` tokens currently and we could be consistent and stick to nouns. Also looks like that's what we're using on the [current central](https://searchfox.org/mozilla-central/rev/9982a76fe95c70f7c2dca2d60bd78e015313924d/toolkit/themes/shared/design-system/tokens-shared.css#94-99).